.eslintrc.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  extends: ['pretty-standard'],
+  plugins: ['prettier', 'import'],
+  rules: {
+    'prettier/prettier': 'error',
+    'import/no-extraneous-dependencies': [
+      'error',
+      {
+        devDependencies: ['**/test/**/*.js'],
+        optionalDependencies: false,
+        peerDependencies: false
+      }
+    ]
+  }
+};
```

Comment:
How about [@gustavnikolaj's `isTTY` trick](https://github.com/unexpectedjs/unexpected/blob/a40ec480bbc3259a20c0cdcb5df05b2b6baac83e/.eslintrc.js#L17-L23)?

Reply:
Fixed it.